### DISCUSSION TITLE
油猴脚本是删除了吗

### DISCUSSION THREAD

**Discussion by pan176** [the1812/Bilibili-Evolved#3178] [提问 / Q&A]
而且我装了脚本后，浏览器左边没有显示插件

**Comment by Geeyun-JY3** [comment]
你这什么具体信息也没有很难帮你判断。

**Comment by pan176** [comment]
![image](https://user-images.githubusercontent.com/41176295/161467288-e0b4b856-e958-46a8-a5e7-3f15ddb8a992.png)

**Reply by the1812** [reply]
早就删了, 你火星了

**Reply by pan176** [reply]
怎么下最新的，求求了

**Reply by Geeyun-JY3** [reply]
都删了一年多了吧……

伸手之前，能不能认先真看一次 README：https://github.com/the1812/Bilibili-Evolved#安装

**Reply by pan176** [reply]
好的谢谢
